### DISCUSSION TITLE
请问为什么抢到票后生成的二维码扫出来是none？

### DISCUSSION THREAD

**Discussion by dxlJill** [mikumifa/biliTickerBuy#138] [提问专区] [answered]
请问为什么抢到票后生成的二维码扫出来是none？

**Comment by mikumifa** [comment]
订单那里有吗

**Comment by dxlJill** [comment, marked as answer]
哦哦看到了，在b站的订单里，话说BW的抢票也能直接生成订单吗




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年6月19日(星期三) 下午3:13
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mikumifa/biliTickerBuy] 请问为什么抢到票后生成的二维码扫出来是none？ (Discussion #138)














 
订单那里有吗
